### ISSUE TITLE
Notebooks pane UX is abysmal

### ISSUE THREAD

**Issue by pshirshov** [streetwriters/notesnook#29]
Currently "Notebooks" pane has strict 3-level hierarchy and navigation experience there feels like a madhouse.

First of all it's not obvious what I'm looking at.  Are these notebooks, "topics" or actually the notes?

Also it's really inconvenient to move the mouse between notes list and "back" button.

Then all the elements in this pane are enormously huge and in case you have even mere 20 notes in a topic you would have to scroll up and down, up and down...  

The "search" button just opens an empty pane without any controls in it.

I would suggest to implement a regular tree-like structure, something similar to Apple Notes and Joplin.

I would happily pay the subscription for this app, I like the idea but at the moment the UX just sucks.

<img width="1141" alt="Screenshot 2021-08-14 at 22 50 43" src="https://user-images.githubusercontent.com/295225/129460999-1c8eeaaa-8a6e-4174-a412-29f2e3f19196.png">
<img width="438" alt="Screenshot 2021-08-14 at 22 52 17" src="https://user-images.githubusercontent.com/295225/129461002-f2a76e55-40a2-466a-9e53-3cefabebd060.png">
<img width="511" alt="Screenshot 2021-08-14 at 22 56 13" src="https://user-images.githubusercontent.com/295225/129461084-e4c7bba0-7871-47b4-8a02-4e012b98aee3.png">

**Comment by thecodrr** [comment]
> First of all it's not obvious what I'm looking at. Are these notebooks, "topics" or actually the notes?

You are actually the first person to bring this to attention. I have put this as a discussion point before the team but we won't take any action until a substantial amount of people are affected by this.

> Then all the elements in this pane are enormously huge and in case you have even mere 20 notes in a topic you would have to scroll up and down, up and down...

You can use the compact mode to see more items in the list. However, sooner or later you will have to scroll up and down - that's literally the whole point of a list. This sounds more "what-i-am-used-to" than an actual issue with the app.

> The "search" button just opens an empty pane without any controls in it.

This was a bug which I have fixed. Thank you for reporting it.

> I would suggest to implement a regular tree-like structure, something similar to Apple Notes and Joplin.
> I would happily pay the subscription for this app, I like the idea but at the moment the UX just sucks.

Thank you so much for considering Notesnook. We are actively working to improve the UX. However, I don't see a tree-like navigation view for Notebooks in the near future. We might add it if enough people ask but no promises.

**Comment by pshirshov** [comment]
Well, while I like the app the half-baked navigation is definitely a dealbreaker for me. I'll wait until you support tree mode and subscribe for Joplin cloud meanwhile.

**Comment by thecodrr** [comment]
> subscribe for Joplin cloud

That's unfortunate but it's entirely up to you. Thank you for trying Notesnook and taking the time out to report all these suggestions and bugs.

**Comment by altano** [comment]
While I don't know if I agree with the severity of the feedback, I do agree with the general sentiment of some of this feedback, fwiw.

> First of all it's not obvious what I'm looking at. Are these notebooks, "topics" or actually the notes?

I think the app could use more breadcrumb trails and iconography. For example, when you click Notebooks and select a Notebook, the top text is the name of the notebook. If you click a topic, the heading remains the name of the Notebook but you get the topic as red text below it. That's not super clear unless you know the app pretty well. Icons for Notebooks and Topics would help things.

> Also it's really inconvenient to move the mouse between notes list and "back" button.

The mouse back/forward button work great in Notesnook.

> Then all the elements in this pane are enormously huge and in case you have even mere 20 notes in a topic you would have to scroll up and down, up and down...

Yeah, the density of notes is extremely low, even in compact view. I can fit a list of 56 pages in OneNote on one screen and in Notesnook Compact View I can see 14. Perhaps the timestamp can be omitted in compact view and perhaps the tags can be next to the title (especially when the title is short).

> The "search" button just opens an empty pane without any controls in it.

This isn't the case for me. Perhaps it has been fixed:

![image](https://user-images.githubusercontent.com/1009/132163626-74517ec3-4979-444c-8cf6-bd336e1da4a8.png)

> I would suggest to implement a regular tree-like structure, something similar to Apple Notes and Joplin.

I don't know if I need a tree, but I honestly miss the arbitrary nesting (deep hierarchy) of OneNote. I also miss being able to see way more about the hierarchy on the screen at once (which Notebook, section, and parent pages are my context), which wasn't distracting because I could press F11 to hide all that in OneNote.

> I would happily pay the subscription for this app, I like the idea but at the moment the UX just sucks.

It doesn't suck for sure. I've tested a million of these notes apps and Notesnook is very high up on UX, even if I do miss a lot of things from OneNote :)
